components/PaymentForm.js

```diff
@@ -164,14 +165,23 @@ function PaymentForm({ onClose = () => null }) {
             ) : (
               <>
                 <IconCards width={27} height={22} />
-                <span className="md:ml-4 ml-2">Donate ₹ {Number(form.amount).toFixed(2)} using </span>
+                <span className="md:ml-4 ml-2">Donate ₹ {getFinalAmount(form.amount)} using</span>
```

Comment:
Sorry if my previous suggestions were not clear. I suggested to call this function only once i.e after the line `21` may be and use the final amount in `openRzp` and here.

Reply:
Sorry, I understood that differently. Will change it push the changes.